src/jekyll/_plugins/wf/WFFeedPage.rb

```diff
@@ -15,7 +15,8 @@
 module Jekyll
 
   class WFFeedPage < LanguagePage
-
+    ## TODO: Handle both atom.liquid and rss.liquid
+    ## TODO: Remove support for feed.liquid
```

Comment:
Is there a need for both rss and atom (I don't know the difference)? Do we need to reference both from the page in the head?

feed.liquid is used by this file and this file only - have you copied the variable names when generating atom and rss?